testing/README.md

```diff
@@ -40,21 +41,23 @@ If you feel that mocha/ sinon/ chai do not do a good job of addressing the way y
 
 - [testdouble](https://github.com/testdouble/testdouble.js) is a great alternative to sinon as a test double library.
 
+If your project is using [ReactJS](https://reactjs.org/) or is built with [Sewing-kit](https://github.com/Shopify/sewing-kit), we recommend using the following tools:
+- [Jest](http://facebook.github.io/jest/) for the test framework. 
+- [Enzyme](http://airbnb.io/enzyme/) for handling spies and mocks.
```

Comment:
Enzyme doesn't handle spies or mocks, it is purely for giving you a nice API for dealing with React components rendered in test. Jest provides its own spying/ mocking utilities that we use.

Reply:
Oops, that's a mistake. I'll clarify.